app/View/Components/DangerButton.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\View\Components;
+
+use Illuminate\View\Component;
+
+class DangerButton extends Component
+{
+    public $type;
+
+    public function __construct(?string $type = 'button')
+    {
+        $this->type = $type;
+    }
+
+    public function render()
+    {
+        return view('components.buttons.danger-button');
+    }
+}
```

Comment:
Does this need to be a class based component? Can just be an anonymous component imo.